src/status_im/wallet/prices.cljs

```diff
@@ -32,10 +32,17 @@
 
 (fx/defn on-update-prices-success
   {:events [::update-prices-success]}
-  [{:keys [db]} prices]
-  {:db (assoc db
-              :prices prices
-              :prices-loading? false)})
+  [{:keys [db]} prices from to]
+  {:db (if (every? true? (for [from-entity from]
+                           (for [to-entity to]
+                             (not (Empty? (get-in db [from-entity to-entity]))))))
```

Comment:
1. `(not-empty ...` can be used instead `(not (Empty? ...` (I doubt that `Empty?` exists, it should have been `empty?`)
2. it is likely that `:prices` is missed in  `get-in`'s `path` `(get-in db [:prices from-entity to-entity])`
3. `for` allows multiple `seq-exprs`, it can be just
```clojure
(for [from-entity from
      to-entity to]
 ...)
```